app/src/main/java/com/infomaniak/drive/utils/PreviewUtils.kt

```diff
@@ -29,16 +29,23 @@ import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
 import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.gson.JsonParser
 import com.infomaniak.drive.MatomoDrive.trackFileActionEvent
 import com.infomaniak.drive.R
 import com.infomaniak.drive.data.models.File
 import com.infomaniak.drive.ui.SaveExternalFilesActivity
 import com.infomaniak.drive.ui.SaveExternalFilesActivityArgs
+import com.infomaniak.drive.ui.fileList.DownloadProgressViewModel.Companion.PROGRESS_COMPLETE
 import com.infomaniak.drive.ui.fileList.preview.BitmapPrintDocumentAdapter
 import com.infomaniak.drive.ui.fileList.preview.PDFDocumentAdapter
+import com.infomaniak.drive.utils.PreviewPDFUtils.PasswordProtectedException
 import com.infomaniak.drive.utils.Utils.openWith
 import com.infomaniak.drive.utils.Utils.openWithIntentExceptkDrive
 import com.infomaniak.lib.core.utils.lightNavigationBar
+import okhttp3.Response
+import java.io.BufferedInputStream
+
+private const val BUFFER_SIZE = 8192
```

Comment:
Why that value ?

Reply:
It's an arbitrary value defined before, I just move it in this PR